docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -109,30 +109,189 @@ Specific kinds of architecture guides include:
 
-### Voice
+Use the first person point of view sparingly and with intention. When overused, the 
+first person narrative can overwhelm the sense of a shared experience and obscure the 
+reader's journey.
 
-The documentation should be technically precise and directed toward a technical
-audience, e.g., application developers, site reliability engineers, and security
-engineering team leads. 
+Don't use “I” or “me” unless it appears in the text of a user interface element.
 
-Even when describing Teleport generally, we should emphasize specific technical
-capabilities over broad statements of benefit. The aim is not to pique the
-audience's interest, but to provide information.
+Avoid using “we” unless you have a really good rationale for it. For example, if you 
+are documenting a recommended setting or practice, using “We recommend...” is ambiguous
+("we" being who exactly?) and potentially dangerous if something goes wrong. If there
+are industry best practices or recommendations, it's okay to indicate that instead of 
+using the royal or editorial "we" in documentation.
+
+### Passive constructions
+
+In spite of the axiom to never use the passive voice, there are situations where a 
+passive sentence structure might be appropriate. Don't twist a sentence into knots just 
+to avoid a passive construction. Passive voice does have its place, but be wary of using it.
+
+When writing about software, it's often tempting to describe what's happening from the 
+code point of view. However, there's almost always a human being with a goal or task to 
+complete who is initiating the activity that the software is executing. If you keep this 
+human presence in mind, your writing will be more dynamic, easier to follow, and more 
+interesting to read.
+
+### Contractions and conversational tone
+
+Contractions are generally acceptable because they give documentation a more natural 
+conversational tone—at least for English speakers. Be conscious of when and why you use 
+contractions.
+
+To keep the tone conversational but concise, adhere to the following common-sense guidelines:
+
+- Use common, well-known words whenever possible.
+
+- Don't use flowery language or literary flourish words and phrases like “and so forth”, 
+  “albeit”, “heretofore”, or “thus”.
+
+- Try to be precise in word choice. For example:
+
+  - Don't use “when”—implying eventuality and time—as interchangeable with “if”, which implies 
+    the possibility of something happening.
+  - Don't use phrases that introduce ambiguity. For example, instead of “When the process completes…” 
+    use “After the process completes…”.
+  - Think carefully about word choices like using “since” (implying a period of time) instead 
+    of “because” (implying cause and result) or using “once” (single occurrence) instead of 
+    “after” (each time).
+
+- Avoid using dead language words and phrases even if they are generally accepted as English 
+  words in practice. For example:
```

Comment:
What's the reasoning for this? English is filled with load words, so this strikes me as somewhat arbitrary and limiting.

Reply:
For clarity, expressing what the dead language or abbreviation means in plain English ensures that everyone understands the intended meaning.
For translation (not an issue for Teleport doc perhaps), using plain English can help translators understand the intended meaning and pick an appropriate term.